eureka_ml_insights/configs/aime.py

```diff
@@ -123,14 +127,132 @@ def configure_pipeline(
             ],
             output_dir=os.path.join(self.log_dir, "eval_report"),
         )
+        
+        # Aggregate the results by a majority vote 
+        self.postevalprocess_comp = EvalReportingConfig(
+            component_type=EvalReporting,
+            data_reader_config=DataSetConfig(
+                DataReader,
+                {
+                 "path": os.path.join(self.evalreporting_comp.output_dir, "metric_results.jsonl"),
+                 "format": ".jsonl",
+                 "transform":RunPythonTransform("df = df.loc[(df.groupby('ID')['model_output'].transform(lambda x: x.mode().iloc[0] if not x.mode().empty else x.iloc[0]) == df['model_output']) | (df['model_output'].isna() & df.groupby('ID')['model_output'].transform(lambda x: x.mode().iloc[0] if not x.mode().empty else x.iloc[0]).isna())].drop_duplicates('ID')")  # compute the majority vote over the model_output. If model_output is null, return null.

```

Comment:
It would help with readability and reusability if you implement a MajorityVoteTransform

Reply:
Added a MajorityVoteTransform